cli/flags/common.go

```diff
@@ -70,6 +72,8 @@ func (commonOpts *CommonOptions) InstallFlags(flags *pflag.FlagSet) {
 	// opts.ValidateHost is not used here, so as to allow connection helpers
 	hostOpt := opts.NewNamedListOptsRef("hosts", &commonOpts.Hosts, nil)
 	flags.VarP(hostOpt, "host", "H", "Daemon socket(s) to connect to")
+	flags.StringVarP(&commonOpts.Context, "context", "c", dockerContext,
```

Comment:
Looks like we have to update the docs for this, or is that done in the other PR?

Reply:
Should do in this one

Reply:
Actually, makes more sense to put all the doc in the following PR so that documentation for commands impacting the default context etc. are documented in sync.

I think once I get 2 LGTM on this one, I will squash, rebase #1501 and close this one so that we merge the full context switch feature at once. Was very usefull for review to split though.